crates/nu-command/tests/commands/cp.rs

```diff
@@ -344,3 +347,42 @@ fn copy_ignores_ansi() {
         assert_eq!(actual.out, "success.txt");
     });
 }
+
+#[test]
+fn copy_file_not_exists_dst() {
+    Playground::setup("cp_test_17", |_dirs, sandbox| {
+        sandbox.with_files(vec![EmptyFile("valid.txt")]);
+
+        let actual = nu!(
+            cwd: sandbox.cwd(),
+            "cp valid.txt ~/invalid_dir/invalid_dir1"
+        );
+        assert!(
+            actual.err.contains("invalid_dir1")
+                && (actual.err.contains("No such file or directory")
+                    || actual
+                        .err
+                        .contains("The system cannot find the path specified"))
```

Comment:
Mhh does this depend on the exact error message from the OS or is this part of the Rust standard library output? While this could be fine in our CI this may break on the machines of developers that use different versions or localized translations.

Reply:
could we just have `invalid_dir!` , I think it should pass .
out just check stderr is not empty and stdout is empty , can we ?

Reply:
I think we should refer to the parts of the message that are defined in this PR (Another reason why the message strings should be slightly different (thanks for the quick land))